lib/phoenix/channel/server.ex

```diff
@@ -149,14 +149,12 @@ defmodule Phoenix.Channel.Server do
   @doc """
   Pushes a reply message to the given pid.
   """
-  def reply(pid, ref, topic, payload)
+  def reply(pid, ref, topic, payload, serializer)
```

Comment:
I thought we would kill this function though because it means the transport is sending itself messages?